config/env.config.js

```diff
@@ -85,15 +91,38 @@ class RedirectPlugin {
                                     return match[1];
                                 }
                                 return null;
-                            };
+                            }
+
+                            function isAllowedUrl(url) {
+                                if (url == null || url.trim().length === 0) {
+                                    return true;
+                                }
+                                
+                                url = decodeURIComponent(url);
+
+                                var validRedirectLocations = ${JSON.stringify(validRedirectLocations)};
+
+                                var matchingLocation = validRedirectLocations.find(function(location) {
+                                    return location.indexOf(url) === 0;
+                                });
+
+                                return matchingLocation ? true : false;
```

Comment:
!!matchingLocation

Reply:
actually this will treat the 0 index as false, we need to test for null

Reply:
Find returns the item, not the index.  So just the null (or undefined) is what we need to check for.
I thought about doing `!!matchingLocation`, but I think it's actually harder to understand.  I never knew about it until I saw Bhargav use it a year ago.  Is there a reason that `!!matchingLocation` is better than the `matchingLocation ? true : false` check that I'm doing?  I could be convinced either way

Reply:
Better yet, using Array.prototype.some:  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some